keyboards/handwired/consolekeyboard/keymaps/via/keymap.c

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2021 Gareth Edwards
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_1,    KC_2,    KC_3,             KC_4,    KC_5,    KC_6,    KC_7,             KC_8,    KC_9,    KC_0,
+        KC_Q,    KC_W,    KC_E,             KC_R,    KC_T,    KC_Y,    KC_U,             KC_I,    KC_O,    KC_P
+    ),
+[1] = LAYOUT(
+        KC_1,    KC_2,    KC_3,             KC_4,    KC_5,    KC_6,    KC_7,             KC_8,    KC_9,    KC_0,
+        KC_Q,    KC_W,    KC_E,             KC_R,    KC_T,    KC_Y,    KC_U,             KC_I,    KC_O,    KC_P
+    ),
+[2] = LAYOUT(
+        KC_1,    KC_2,    KC_3,             KC_4,    KC_5,    KC_6,    KC_7,             KC_8,    KC_9,    KC_0,
+        KC_Q,    KC_W,    KC_E,             KC_R,    KC_T,    KC_Y,    KC_U,             KC_I,    KC_O,    KC_P
+    ),
+[3] = LAYOUT(
+        KC_1,    KC_2,    KC_3,             KC_4,    KC_5,    KC_6,    KC_7,             KC_8,    KC_9,    KC_0,
+        KC_Q,    KC_W,    KC_E,             KC_R,    KC_T,    KC_Y,    KC_U,             KC_I,    KC_O,    KC_P
+    ),
+	};
```

Comment:
If the key on the higher layer is the same as the layer below, just set it to `KC_TRNS` (or `_______` which is a synonym). That way it will "fall through" to the lower numbered layer.

Reply:
Have updated to make keys transparent on higher layers on via keymap